src/Component/Taskbar/Tasks.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import {bindActionCreators} from 'redux'
+import {addTask}  from '../../Actions/ActList'
+class Tasks extends React.Component{ 
+    render()
+    {
+        return(
+            <div> 
```

Comment:
You dont need a extra div here.
Since react needs one child to be returned you can use React.Fragment or (<>) to club the under one. It does not create a extra DOM element. Read about it on react documentation page.